infra/registry/main.tf

```diff
@@ -0,0 +1,37 @@
+
+resource "openstack_compute_servergroup_v2" "servergroup" {
+  name     = "servergroup"
+  policies = ["anti-affinity"]
+}
+
+variable "instance_count" {
+  type    = number
+  default = 2
+}
+
+resource "openstack_compute_keypair_v2" "my-cloud-key" {
+  name       = "my-key"
+  public_key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIhZECaYYf3DmbgyHQJWyJqTIqzQSbF87JUTX5eL/mRm"
+}
+
+resource "openstack_compute_instance_v2" "instance" {
+  count = var.instance_count
+
+  name            = "instance-${count.index + 1}"
+  image_name      = "Mekstack Ubuntu 22.04.3 LTS"
+  flavor_name     = "m2s.small"
```

Comment:
Pass image name and flavor from root module like in infra/sneedaas or infra/dnosha

Reply:
Is it possible to define and pass names from root module without inputting them from keyboard?